nginx.conf

```diff
@@ -1,9 +1,23 @@
-# for Translations
+index index.html index.htm index.php;
+error_page 404 /404.php;
+
 location / {
     rewrite "^/([a-z]{2}(?:_(?:[A-Z]{2}|[A-Z][a-z]+))?)/(.*)$" /$2?lang=$1 last;
     try_files $uri $uri.html $uri/ @extensionless-php;
 }
 
+location @extensionless-php {
+    rewrite ^(.*)$ $1.php last;
+}
+
+location ~ \.php$ {
```

Comment:
`location ~ \.php$ {` ?

I would have thought that applies to all Nginx installations.